src/Conference/AddConferenceForm.test.js

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import { Router, MemoryRouter } from "react-router-dom";
+import { act } from "react-dom/test-utils";
+import { mount, configure } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import { createMemoryHistory } from "history";
+import { WrappedAddConferenceForm as AddConferenceForm } from "./AddConferenceForm";
+import { createNewConference } from "../api";
+
+configure({ adapter: new Adapter() });
+
+jest.mock("../api", () => ({
+  createNewConference: jest.fn()
+}));
+
+describe("submitForm", () => {
+  let ev;
+  beforeEach(() => {
+    ev = { preventDefault: jest.fn() };
+  });
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  const mockCookie = {
+    sessionToken: {
+      userId: 1,
+      token: "3ecb9d1d-863f-4207-b076-d868e6544c3b"
+    }
+  };
+
+  it("should send axios request", async () => {
+    expect.assertions(2);
+
+    const apiReturnValue = Promise.resolve(200);
+    createNewConference.mockImplementation(() => apiReturnValue);
+
+    let wrapper;
+    act(() => {
```

Comment:
These shouldnt need the act wrapper?

act should only be used when there is active change in the state, usually to test mount events etc. rather than pure setup?